packages/react-core/src/next/components/Modal/examples/Modal.md

```diff
@@ -0,0 +1,141 @@
+---
+id: Modal
+section: components
+cssPrefix: pf-v5-c-modal-box
+propComponents: ['Modal', 'ModalBoxBody', 'ModalBoxHeader', 'ModalBoxFooter']
+ouia: true
+beta: true
+---
+
+import WarningTriangleIcon from '@patternfly/react-icons/dist/esm/icons/warning-triangle-icon';
+import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon';
+import BullhornIcon from '@patternfly/react-icons/dist/esm/icons/bullhorn-icon';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import formStyles from '@patternfly/react-styles/css/components/Form/form';
+
+## Examples
+
+### Basic modals
+
+Basic modals give users the option to either confirm or cancel an action. To flag an open modal, use the `isOpen` property. To execute a callback when a modal is closed, use the `onClose` property.
```

Comment:
```suggestion
Basic modals give users the option to either confirm or cancel an action. 

To flag an open modal, use the `isOpen` property. To execute a callback when a modal is closed, use the `onClose` property.

A modal must have a `<ModalBoxBody>`, containing the main content of the modal. The `<ModalBoxHeader>` and `<ModalBoxFooter>` components are not required, but are typically used to display the modal title and any button actions, respectively. 
```
If we think this is a valuable callout to have here - vs if this is stated in the props table 🤷‍♀️

Reply:
I like this description.